srcCpp/perftest_publisher.cxx

```diff
@@ -1111,6 +1117,8 @@ int perftest_cpp::Subscriber()
     fprintf(stderr,"Waiting for data ...\n");
     fflush(stderr);
 
```

Comment:
Add a comment explaining briefly what this does.

Reply:
ok